tests/i18n/test_locale.py

```diff
@@ -221,3 +237,18 @@ def test_set_locale(self):
                     self.assertFalse(True)
```

Comment:
Maybe this would be more appropriate here https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertNotIn